replace/modernized/03-chapter-3.adoc

```diff
@@ -540,45 +593,50 @@ by pleading the devil`'s cause so frequently,
 which is to keep people in sin for term of life, which if he can by any means do,
 he knows they must come to him at the last, because the Son of God has spoken it, Luke 8:27.
 
+[.numbered]
 16+++.+++ It is contrary to all these plain Scripture proofs, John 8:1,34 John 3:4,
 1 John 3:8, He that commits sin is of the devil, etc.
 
+[.numbered]
 17+++.+++ To say that none can be perfect,
 does make void the apostle`'s prayers for the Hebrews, when he parted from them;
 who prayed, that the God of peace,
 that brought again from the dead our Lord Jesus Christ, that great Shepherd of the sheep,
 through the blood of the everlasting covenant,
 make you perfect in every good work to do his will, Heb. 13:20-21.
 
+[.numbered]
 18+++.+++ To say that none can be perfect on earth, makes Paul a liar, who said to the Hebrews,
 But you are come to Mount Zion, unto the city of the living God, the heavenly Jerusalem,
 to the innumerable company of angels, Heb. 12:22.
 To the general assembly and church
 of the first-born which are written in heaven, and to God the Judge of all,
 and to the spirits of just men made perfect, verse 23.
 
+====
+
 But it may be objected,
 that there is not a just man upon earth that does good and sins not.
 
-Answer.
+_Answer:_
```

Comment:
these answers and objections (there are several) all need to be marked as: [.discourse-part]